src/main/java/com/bymarcin/openglasses/surface/Widget.java

```diff
@@ -63,10 +63,10 @@ public void readFromNBT(NBTTagCompound nbt){
 	public abstract IRenderableWidget getRenderable();
 
 	public boolean isVisible() {
-		return isVisable;
+		return isVisible;
```

Comment:
These were likely here to keep the "typo compatibility" - if we control everything that calls this we can fix it (and remove these)

Reply:
are attributes public or private by default?  Does anything use the attribute directly? If they're private by default, or nothing uses them, it's probably fine.

Reply:
Quick google indicates it's "package private" with no keyword specified.

Reply:
It still have public "setVisable" setter and ingame getter "isVisable", so yeah, it will brake some things
Not sure we should keep backward compatibility that hard tho. I doubt this repo was used or modified by someone and from ingame perspective things brake with each update anyway lol
This typo is extremely irritating for me xD